pilot/pkg/networking/grpcgen/lds.go

```diff
@@ -153,6 +154,21 @@ func buildFilterChain(nameSuffix string, tlsContext *tls.DownstreamTlsContext) *
 			Name: "inbound-hcm" + nameSuffix,
 			ConfigType: &listener.Filter_TypedConfig{
 				TypedConfig: util.MessageToAny(&hcm.HttpConnectionManager{
+					RouteSpecifier: &hcm.HttpConnectionManager_RouteConfig{
+						// https://github.com/grpc/grpc-go/issues/4924
+						RouteConfig: &route.RouteConfiguration{
+							Name: "inbound",
+							VirtualHosts: []*route.VirtualHost{{
+								Domains: []string{"*"},
+								Routes: []*route.Route{{
+									Match: &route.RouteMatch{
+										PathSpecifier: &route.RouteMatch_Prefix{Prefix: "/"},
+									},
+									Action: &route.Route_NonForwardingAction{},
```

Comment:
was this a bug before?

Reply:
Yes, see the issue linked in the code for info. A bug in both gRPC and Istio